src/sentry/notifications/helpers.py

```diff
@@ -847,6 +851,32 @@ def user_has_any_provider_settings(
     return False
 
 
+def is_double_write_enabled(user_id: Optional[int] = None, team_id: Optional[int] = None):
+    from sentry.models import Team
+    from sentry.services.hybrid_cloud.organization_mapping.serial import (
+        serialize_organization_mapping,
+    )
+
+    if team_id is not None:
+        # this only works in a region silo so if this is called from a
+        # control silo we need to figure out what to do
+        orgs = [Team.objects.get(id=team_id).organization]
+    elif user_id is not None:
+        # done in a control silo so use the maping
+        org_ids = OrganizationMemberMapping.objects.filter(user_id=user_id).values_list(
+            "organization_id", flat=True
+        )
+        orgs = list(
+            map(
+                serialize_organization_mapping,
+                OrganizationMapping.objects.filter(organization_id__in=list(org_ids)),
+            )
+        )
```

Comment:
Does this function run in both silos with different parameters? You won't be able to access teams in control silo.

Reply:
@markstory i guess it has to run in the control silo, I'm getting around this by passing in `organization_id_for_team` so we can look up the feature flag that way.